Booting/linux-bootstrap-1.md

```diff
@@ -1,29 +1,31 @@
+If you have read my previous [blog posts](http://0xax.blogspot.com/search/label/asm), you can see that some time ago I started to get involved with low-level programming. I wrote some posts about x86_64 assembly programming for Linux. At the same time, I started to dive into the Linux source code. It is very interesting for me to understand how low-level things work, how programs run on my computer, how they are located in memory, how the kernel manages processes and memory, how the network stack works on low-level and many many other things. I have therefore decided to write another series of posts about the Linux kernel for **x86_64**.
 
-Note that I'm not a professional kernel hacker, and I don't write code for the kernel at work. It's just a hobby. I just like low-level stuff, and it is interesting for me to see how these things work. So if you notice anything confusing, or if you have any questions/remarks, ping me on twitter [0xAX](https://twitter.com/0xAX), drop me an [email](anotherworldofworld@gmail.com) or just create an [issue](https://github.com/0xAX/linux-insides/issues/new). I appreciate it. All posts will also be accessible at [linux-insides](https://github.com/0xAX/linux-insides) and if you find something wrong with my English or post content, feel free to send pull request.
+Please note that I'm not a professional kernel hacker, and I don't write code for the kernel at work. It's just a hobby; I like low-level stuff, and it is interesting for me to see how these things work. If you notice anything confusing, or if you have any questions/remarks, ping me on twitter [0xAX](https://twitter.com/0xAX), drop me an [email](anotherworldofworld@gmail.com) or just create an [issue](https://github.com/0xAX/linux-insides/issues/new). I would really appreciate it. All posts will also be accessible at [linux-insides](https://github.com/0xAX/linux-insides) and if you find something wrong with my English or post content, feel free to send pull request.
 
 
 *Note that this isn't official documentation, just learning and sharing knowledge.*
 
 **Required knowledge**
 
-* Understanding C code
-* Understanding assembly code (AT&T syntax)
+* An understanding of C code
+* An understanding of assembly code (AT&T syntax)
 
-Anyway, if you just started to learn some tools, I will try to explain some parts during this and following posts. Ok, little introduction finished and now we can start to dive into kernel and low-level stuff.
+If you just started to learn some tools, I will try to explain some parts during this and the following posts. Ok, our little introduction is finished and now we can start to dive into th kernel and low-level stuff.
```

Comment:
>Anyway, if you just started

The colloquialisms broke my flow when I was reading it over - I'd hate to take away from 0xAX's style though. Just a judgement call. 

>this and the following

This is correct. 'Following' means *in response to* or *as a result of* whereas 'the following' is used to introduce a noun or collective noun.

>the kernel

will correct this